templates/common/gcp/units/gcp-routes-service.yaml

```diff
@@ -0,0 +1,19 @@
+name: "gcp-routes.service"
+enabled: true
+contents: |
+  [Unit]
+  Description=Update GCP routes for forwarded IPs.
+  ConditionKernelCommandLine=|ignition.platform.id=gce
+  ConditionKernelCommandLine=|ignition.platform.id=gcp
+  After=network.target
```

Comment:
Is it recommended to use `network.target` or `network-online.target` in Fedora?

Reply:
https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/ covers this...I think `network-online.target` is more right but the script retries anyways.

Reply:
since we already have `network.target` in vsphere UPI scripts I'll keep `network.target` - and replace these later